nodebuilder/p2p/network.go

```diff
@@ -46,10 +46,10 @@ var networksList = map[Network]struct{}{
 	Private: {},
 }
 
-// networkAliases is a strict list of all known long-standing networks
+// NetworkAliases is a strict list of all known long-standing networks
 // mapped from the string representation of their *alias* (rather than
 // their actual value) to the Network.
-var networkAliases = map[string]Network{
+var NetworkAliases = map[string]Network{
```

Comment:
networkList is private, but aliases are not 🤔 

My suggestion is to make `Validate` check for aliases as well to keep aliases encapsulated

Reply:
Dunk, I missed that we actually need the right Network to be swapped so that we determine the path correctly, and this is why you make Validate return it, which is also a bit weird. Another thought I had is to make the path to take aliases instead of real network names. That's nice because there will never be two Arabicas simultaneously, but it is slightly breaking. This, though, would allow us to make Validate not return anything, as we use aliases everywhere besides where real chain-id is actually needed.

Reply:
I think that adds more work than is worth.

Reply:
Agreed, just don't like that Validate returns new network